src/main/java/kitchenpos/ui/TableGroupRestController.java

```diff
@@ -26,7 +32,7 @@ public ResponseEntity<TableGroup> create(@RequestBody final TableGroup tableGrou
 
     @DeleteMapping("/api/table-groups/{tableGroupId}")
     public ResponseEntity<Void> ungroup(@PathVariable final Long tableGroupId) {
-        tableGroupService.ungroup(tableGroupId);
+        tableGroupService.ungroup(UnGroupRequest.of(tableGroupId));
```

Comment:
Real Questoin) Is there any verb 'ungroup' in English???

Reply:
Yeah, there is actual word "ungroup", not frequently used in verbal though.